volontulo/templates/common/footer.html

```diff
@@ -19,6 +19,7 @@
             <h3>Organizacje</h3>
             <ul>
                 <li><a href="{% url 'organizations_list' %}">Lista organizacji</a></li>
+                <li><a href="{% url 'organizations_create' %}">Załóż organizację</a></li>
```

Comment:
As we use urganization_view, organization_form and organizations_list, so it could be nice to make organization_create link.